### PR TITLE
Sysdig team access rework - Merge sysdig container and host team

### PR DESCRIPTION
**Pull request by ShellyXueHan**
### background:
we used to create two sysdig teams per project set: `abc-team` and `abc-persistent-volume-team`. That was because PVC metrics are coming from kubelet and thus under a different scope. Recently, sysdig has updated their datastore, where kubelet metrics are no more exclusive from container scoped teams. So from now on we only need one team per project set.

I've reworked the default dashboard to contain CPU/memory/storage resource usages. Also assigned the dashboard to be owned by a service account instead of a team member to prevent mismatch user issues.

### changes:
- added the github actions
- remove team and dashboard creation task for PVC specific metric
- added new dashboard template for resource usage monitoring
- created a service account that will be hte owner of the default dashboard template that we created for teams

### note:
- doc updates will be in another PR, coming soon!

### CODE REVIEW

**Review by StevenBarre — APPROVED** [review]
Do existing teams need any migration steps?

**Comment by ShellyXueHan** [comment]
@StevenBarre i'll bulk upload the new dashboard template for all teams, so if they don't have any other setup, there is no need for migration. But I do plan to leave the migration of existing dashboards to teams themselves as they might have alerts and notification channels from the PVC team. Will work on some documents next step!

**Comment by ShellyXueHan** [comment]
tested in klab, all good and ready to merge now!

File: .github/workflows/sysdig-teams-operator-build-prod.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      id: branchref
+    - uses: actions/checkout@v1
+      with:
+        ref: ${{ github.ref }}
+    - name: Login to DockerHub Registry
+      run: echo ${{ secrets.DOCKERHUB_PASSWORD }} | docker login -u ${{ secrets.DOCKERHUB_USERNAME }} --password-stdin
+    - name: Build the tagged Docker image
+      run: cd operator/sysdig-monitor && docker build . --file Dockerfile --tag bcdevopscluster/sysdig-teams-operator:prod
+    - name: Push the tagged Docker image
+      run: docker push bcdevopscluster/sysdig-teams-operator:prod
```
Comment (IanKWatts) [inline .github/workflows/sysdig-teams-operator-build-prod.yaml:26]: At some point, these builds should be going into Artifactory instead of DockerHub.

Comment (ShellyXueHan) [reply .github/workflows/sysdig-teams-operator-build-prod.yaml:26]: yes totally! i got a ticket to move from docker to artifactory. Gotta figure out how to do that on github actions first and the rest should be pretty straight forward!

Comment (StevenBarre) [reply .github/workflows/sysdig-teams-operator-build-prod.yaml:26]: Can I get a demo/walkthrough once you figure it out?

Comment (ShellyXueHan) [reply .github/workflows/sysdig-teams-operator-build-prod.yaml:26]: sure. if pushing to artifactory directly doesn't work then plan B is probably just to trigger a build in namespace.

**Pull request merged**